build.rs

```diff
@@ -19,10 +19,25 @@ fn main() {
 
     assert!(status.success(), "make failed");
 
-    println!("cargo:rustc-link-search={}", libbpf_dir.display());
+    let status = process::Command::new("make")
+    .arg("libxdp")
+    .current_dir(&xdptools_dir)
+    .status()
+    .expect("could not execute make");
+
+    assert!(status.success(), "make libdxp failed");
```

Comment:
This is duplicated, right?

Reply:
Ups, seems you're correct. I cherry-picked that from my other branch and it seems something went wrong. Is fixed now.